doc/nrf/libraries/modem/modem_slm.rst

```diff
@@ -0,0 +1,101 @@
+.. _lib_modem_slm:
+
+Modem SLM
+#########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Modem SLM library exposes the AT command interface of the Serial LTE Modem (:ref:`slm_description`) for external devices over a serial interface.
+This library is for applications running on external MCU built with nRF Connect SDK and connected to nRF9160 SiP through UART.
```

Comment:
```suggestion
This library is for applications running on external MCU built with |NCS| and connected to nRF9160 SiP through UART.
```
Convention, in case the name ever needs to change.